### DISCUSSION TITLE
Achievement Categories and Accessibility

### DISCUSSION THREAD

**Discussion by LadyLumina** [RetroAchievements/RAWeb#1590] [Ideas]
From going through RetroAchievements for a while, I noticed quite a few potential flaws (or rather places that could use quite a few improvements), though not critical, these could be drastically improved for the ease of playing and access.

The first thing to note is the inconvenience of Subsets of games, especially in the same playthrough. This causes one to require swapping in and out from game versions just for individual achievements or at worst, do an entirely new playthrough or few just for them. These should be loaded simultaneously with the others in a separate category. For instance, going into the achievements tab on RetroArch could show the lists of types similar to how PlayStation handles DLC trophies: outside the main set (excluded from the Platinum trophy which is the equivalent of a Mastery Badge here) and the additional ones in their own category. 

This might be controversial, though I would wish to bring up the frequent topic of "frustrating" achievements as expressed by quite a few. I always stand by the opinion that a single achievement can ruin an entire set, whether it is here, PlayStation, Xbox, Steam, or whatever it is. This is due to the fun factor being pushed into a category of discomfort or stress that deviates from the core experience, catering to a very small portion of the player base whilst causing passive or direct stress to other players, especially with the rising trend of full completion. As such, I would believe achievements should be allowed to be sorted (and enabled/disabled from the main to bonus sets) so the main set would usually include an experience directly correlating to a natural (full completion 100%) playthrough whilst other more hardcore sets a player would not normally partake in years ago during the original release would be sorted into another category akin to challenge and bonus sets but rather their own thing. Such as sort all the collectibles, events, stage progression, and so forth. But have the sub category available for Developers for the likes of our already known Bonus sets, and things such as no hit runs, highest difficulty playthroughs, and so forth. That way, the achievement lists will be not too daunting for players whilst Developers will have whatever freedom they want for achievement categories and requirements.

TLDR: Game files in-emulator toggle for achievement set loading. Sorting pre-existing sets (similar to the promotion and demotion of unofficial achievements) into subsets for the ease of the player base.

**Comment by Tsearo** [comment]
Playing subsets along the core set has been discussed and is one of the most highly requested features, some discussion on it can be see here 
https://github.com/RetroAchievements/RAWeb/discussions/707#discussioncomment-3213349 where the main thing that needs to be figured out is how to allow players to select which subsets to enable/disable

**Reply by LadyLumina** [reply]
Oh, silly me. Ehe. I suppose I shall pop in there then. Thank you~.
